staging/src/github.com/kubeedge/mapper-framework/pkg/util/parse/grpc.go

```diff
@@ -106,17 +106,17 @@ func buildPropertiesFromGrpc(device *dmiapi.Device) []common.DeviceProperty {
 		var dbconfig common.DBConfig
 		var pushMethod []byte
 		var pushMethodName string
-		if pptv.PushMethod != nil && pptv.PushMethod.DBMethod != nil {
+		if pptv.PushMethod != nil && pptv.PushMethod.DbMethod != nil {
```

Comment:
This should be `pptv.PushMethod.DBMethod` . This problem has indeed appeared before. I don’t know why the automatically generated file is always `DbMethod`, but it seems that `DBMethod` is more in line with the naming requirements. You can manually replace the automatically generated `DbMethod` with `DBMethod` in the generated api.pb.go.

Reply:
This is because of the proto file generation rules, unless it is changed to DBMethod DBMethod = 3;

![image](https://github.com/user-attachments/assets/ae2abe3e-64ee-4756-a137-3524bc673c77)